magnum_cluster_api/resources.py

```diff
@@ -419,6 +419,19 @@ def get_object(self) -> objects.KubeadmControlPlaneTemplate:
                     "template": {
                         "spec": {
                             "kubeadmConfigSpec": {
+                                "format": "ignition",
+                                "ignition": {
+                                    "containerLinuxConfig": {
+                                        "additionalConfig": "storage:\n  links:\n  - path: /etc/systemd/system/kubeadm.service.wants/containerd.service\n    target: /usr/lib/systemd/system/containerd.service\n"
```

Comment:
I think you can use `textwrap.dedent` and have a cleaner multiline string here :)